feature: add next.js, react quickstarts (#1093)
knocklabs/docs: content/getting-started/quick-start/nextjs.mdx

---
title: Get started with Knock
description: Quickly get up and running with Knock and Next.js.
tags: ["getting started"]
section: Getting started
---

> In this guide, you'll integrate Knock's `NotificationFeed` component in Next.js and send your first notification using a server action.

Reviewer: Like what Scoti called out above, we'll want to remove our use of "guide" here
```suggestion
This page covers how to integrate Knock's `NotificationFeed` component in Next.js and send your first notification using a server action.
```

<img className="rounded-md mx-auto border border-gray-200" alt="use commit button to commit your changes" style={{marginBottom: "12px"}} src="/images/quick-start/commit.png"/>
        You do this using the `Commit` button  in the header of the workflow canvas. You can review your changes using a diff viewer and leave a meaningful commit message about what has changed.
        <img className="rounded-md mx-auto border border-gray-200" alt="diff your changes and leave a message" style={{marginBottom: "12px"}} src="/images/quick-start/commit-diff.png"/>
      </Accordion>
    </AccordionGroup>
    Now we're ready to trigger our workflow via the Knock API. You can also learn more about workflows and channels in Knock via our [documentation on designing workflows](/send-notifications/designing-workflows).

  </Step>
  <Step title="Create a server action">
    Now, you'll trigger your workflow to notify your user using a server action. Run the following command to create a file named `trigger.action.ts` in the `app` directory:

>     ```bash 

Reviewer 2: ```suggestion
    ```bash title="Create a file named trigger.action.ts"
```
